Modules/getpath.c

```diff
@@ -1291,21 +1291,26 @@ calculate_read_pyenv(PyCalculatePath *calculate)
 }
 
 
+#define _PPCAT_NX(A, B) A ## B
+#define _PPCAT(A, B) _PPCAT_NX(A, B)
+#define _STRINGIZE_NX(A) #A
+#define _STRINGIZE(A) _STRINGIZE_NX(A)
+
 static PyStatus
 calculate_zip_path(PyCalculatePath *calculate)
 {
     PyStatus res;
 
     /* Path: <PLATLIBDIR> / "python00.zip" */
-    wchar_t *path = joinpath2(calculate->platlibdir_macro, L"python000.zip");
+    wchar_t *path = joinpath2(calculate->platlibdir_macro, L"python" _STRINGIZE(_PPCAT(PY_MAJOR_VERSION, PY_MINOR_VERSION)) L".zip");
```

Comment:
Would the following code work?
```suggestion
    wchar_t *path = joinpath2(calculate->platlibdir_macro, L"python" Py_STRINGIFY(PY_MAJOR_VERSION) Py_STRINGIFY(PY_MINOR_VERSION) L".zip");
```

Reply:
Yes, it works! I was not aware of `Py_STRINGIFY`